README.md

```diff
@@ -1,2 +1,22 @@
-# WebEngDus
- Everything you need to know about how we run the Web Engineering DUS Meetup.
+# WebEngDUS
+
+Everything you need to know about how we run the *Web Engineering DUS Meetup*.
+
+## Where i can find you?
+
+The *Web Engineering Düsseldorf* is available at several places:
+
+* [Web Engineering Düsseldorf at Meetup](https://www.meetup.com/Web-Engineering-Duesseldorf/)
+* [@WebEngDUS at twitter](https://twitter.com/WebEngDUS)
+* [WebEngDUS at GitHub](https://github.com/WebEngDUS)
+
+## Where can i get more information or provide feedback?
+
+If you want to get more information and you have a specific question, feel free to [open an issue at our GitHub repository](https://github.com/WebEngDUS/WebEngDUS/issues/new).
+We will be happy to answer it.
+
+When you want to provide us feedback to this content, to the way we run the meetup or to the meetup itself, [open an issue at our GitHub repository](https://github.com/WebEngDUS/WebEngDUS/issues/new) is also the way to go.
```

Comment:
"**If** you want to provide us feedback **regarding** this content, the way we run the meetup or anything else, open**ing** and issue ..."